edx_proctoring/static/proctoring/js/views/proctored_exam_add_allowance_view.js

```diff
@@ -160,38 +205,37 @@ edx = edx || {};
                 });
             }
         },
-        selectExamAtIndex: function(index) {
-            var selectedExam = this.proctored_exams[index];
-
-            if (selectedExam.is_proctored) {
-                // Selected Exam is a Proctored or Practice-Proctored exam.
-                if (selectedExam.is_practice_exam) {
-                    $('#exam_type_label').text(gettext('Practice Exam'));
-                } else {
-                    $('#exam_type_label').text(gettext('Proctored Exam'));
-                }
-
-                // In case of Proctored Exams, we hide the Additional Time label and show the Allowance Types Select
-                $('#additional_time_label').hide();
-                $('select#allowance_type').val('additional_time_granted').show();
-            } else {
-                // Selected Exam is a Timed Exam.
-                $('#exam_type_label').text(gettext('Timed Exam'));
-
-                // In case of Timed Exams, we show the "Additional Time" label and hide the Allowance Types Select
-                $('#additional_time_label').show();
-                // Even though we have the Select element hidden, the backend will still be using
-                // the Select's value for the allowance type (key).
-                $('select#allowance_type').val('additional_time_granted').hide();
-            }
-            this.updateAllowanceLabels('additional_time_granted');
+        selectExamAtIndex: function(examID, examName) {
+            console.log(examID);
```

Comment:
Remember to update these console logs to either a reasonable console message, or complete remove them.